i18n/en.yaml

```diff
@@ -7,6 +7,22 @@
   translation: "Star"
 - id: github.stars
   translation: "Stars"
+- id: linkedin.star
+  translation: "Star"
+- id: linkedin.stars
+  translation: "Stars"
+- id: twitter.star
+  translation: "Star"
+- id: twitter.stars
+  translation: "Stars"
+- id: facebook.star
+  translation: "Star"
+- id: facebook.stars
+  translation: "Stars"
+- id: custom.star
+  translation: "Star"
+- id: custom.stars
+  translation: "Stars"
```

Comment:
Yeah button text should probably not be using the key "star" as this only makes sense for Github starring. Maybe a more neutral `id: facebook.button_text` could work as well as the translation being being `Facebook`.

Reply:
Same for the other languages.

Reply:
OK. I'll incorporate those changes as well.